buildlib/tools/nvhpc-pgi-compiler

```diff
@@ -0,0 +1,52 @@
+#%Module1.0
```

Comment:
1. why we put modulefile in ucx and not in rdmz env?
2. move it to buildlib/modulefiles